var/spack/repos/builtin/packages/uqtk/package.py

```diff
@@ -38,8 +48,21 @@ def cmake_args(self):
         lapack_libs = spec['lapack'].libs.joined(';')
         blas_libs = spec['blas'].libs.joined(';')
 
-        return [
+        args = [
             '-DCMAKE_SUNDIALS_DIR={0}'.format(spec['sundials'].prefix),
             '-DLAPACK_LIBRARIES={0}'.format(lapack_libs),
-            '-DBLAS_LIBRARIES={0}'.format(blas_libs)
+            '-DBLAS_LIBRARIES={0}'.format(blas_libs),
         ]
+
+        if self.spec.variants['pyuqtk'].value:
+            args.extend(['-DPyUQTk=ON'])
```

Comment:
You can rewrite all of the above as:
```python
args = [
    self.define('CMAKE_SUNDIALS_DIR', spec['sundials'].prefix),
    self.define('LAPACK_LIBRARIES', lapack_libs),
    self.define('BLAS_LIBRARIES', blas_libs),
    self.define_from_variant('PyUQTk', 'pyuqtk')
]
```

Reply:
Yes, much better, thanks. I wasn't sure if this would work since UQTk expects `-DPyUQTk=ON`.